vlib/v/checker/tests/comptime_value/use_flag_comptime_values.vv

```diff
@@ -0,0 +1,15 @@
+// This file should pass if compiled/run with:
+// v -cv my_f64=2.0 -cv my_int=3 -cv my_string="a four" -cv my_bool=true -cv my_char=g run vlib/v/checker/tests/comptime_value/use_flag_comptime_values.vv
+const my_f64 = $compile_value('my_f64', 1.0)
+const my_int = $compile_value('my_int', 2)
+const my_string = $compile_value('my_string', 'three')
+const my_bool = $compile_value('my_bool', false)
+const my_char = $compile_value('my_char', `f`)
+
+fn main() {
+	assert my_f64 == 2.0
+	assert my_int == 3
+	assert my_string == 'a four'
+	assert my_bool == true
+	assert my_char == `g`
+}
```

Comment:
move this file in `vlib/v/gen/c/testdata/`, then add a comment at the top:
```
// vtest vflags: -cv my_f64=2.0 -cv my_int=3 -cv my_string="a four" -cv my_bool=true -cv my_char=g
```
then add in the same folder, a file named: `use_flag_comptime_values.out` .

Then, it will be tested automatically by `v vlib/v/gen/c/coutput_test.v` which is run by `v test-self`, and there will be no need for the separate .yml job.

It will be also easier to test locally.